addons/binding/org.openhab.binding.opensprinkler/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,501 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="opensprinkler"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- The stand alone OpenSprinkler thing type -->
+    <thing-type id="opensprinkler">

```

Comment:
to avoid "opensprinkler:opensprinkler" in all UIDs, I would suggest to change the type id to "http" and "pi" for the two thing types. This makes the UIDs better readable.